actiontext/test/unit/content_test.rb

```diff
@@ -164,6 +164,12 @@ class ActionText::ContentTest < ActiveSupport::TestCase
     assert_equal trix_html, content_from_html(html).to_trix_html.strip
   end
 
+  test "sanitizes attachment markup for Trix if content exists but is empty" do
+    html = '<action-text-attachment sgid="123" content=""></action-text-attachment>'
+    trix_html = '<figure data-trix-attachment="{&quot;sgid&quot;:&quot;123&quot;}"></figure>'
+    assert_equal trix_html, content_from_html(html).to_trix_html.strip
+  end
+
```

Comment:
Could you add a test to test no content attribute is added if it is empty after sanitization?
```suggestion
  test "removes content attribute if it's value is empty after sanitization" do
    html = '<action-text-attachment sgid="123" content="<script>alert('hi!')</script>"></action-text-attachment>'
    trix_html = '<figure data-trix-attachment="{&quot;sgid&quot;:&quot;123&quot;}"></figure>'
    assert_equal trix_html, content_from_html(html).to_trix_html.strip
  end
```